apps/system/js/statusbar.js

```diff
@@ -431,6 +440,12 @@
         case 'cardviewshown':
         case 'cardviewclosed':
           return 'cardview';
+        case 'appwillopen':
+        case 'stackchanged':
+          return 'appopening';
+        case 'appwillclose':
+        case 'stackchanged':
```

Comment:
Yeah, you are right... I was using appopening/opened, appclosing/closed, but apparently there is an issue now and the appclosed event is not being triggered, so I replaced it with stackchanged and I commit it wrong. The idea is having the 3 events (opening/closing/stackchanged) grouping a apptransitioning event. How does that sound?